src/Twig/RichEditorExtension.php

```diff
@@ -123,7 +139,7 @@ private function renderElements(array $elements): string
      *
      * @return string
      */
-    private function renderElement(array $element): string
+    public function renderElement(array $element): string
```

Comment:
Why it becomes public ?

Reply:
To be able to use them from filter/function